### ISSUE TITLE
Any way to reveal to 60 & 260?

### ISSUE THREAD

**Issue by dan55304** [John-Lluch/SWRevealViewController#207]
I have side menus with icons.  I have the reveals set up:
    self.revealViewController.rearViewRevealWidth = 60;
    self.revealViewController.rearViewRevealOverdraw = 260;
    self.revealViewController.rightViewRevealWidth = 60;
    self.revealViewController.rightViewRevealOverdraw = 260;

Only my icon shows when the rear or right menu is revealed.  So far so good.  What I'd like is for the drag gesture to lock at 260.  Setting bounce back to NO doesn't stop the reveal from reverting to 60 pixels.

Setting stableDragOnOverdraw to YES will let me drag the front VC totally off the display but that's not what I want.  I'd like it to either stop where the user dragged to or go to my overdraw size of 260.

In other words, I'd like the reveal to be 0, 60, or 260.  Is that possible?
